proposals/lip-0052.md

```diff
@@ -530,31 +521,23 @@ def verify(trs: Transaction) -> None:
     senderAddress = sha256(trs.senderPublicKey)[:LENGTH_ADDRESS]
     nftID = trsParams.nftID
     receivingChainID = trsParams.receivingChainID
+    recipientAddress = trsParams.recipientAddress
+    data = trsParams.data
+    messageFee = trsParams.messageFee
     messageFeeTokenID = trsParams.messageFeeTokenID
 
-    if receivingChainID == OWN_CHAIN_ID:
-        raise Exception("Receiving chain cannot be the sending chain.")
-
-    if NFTStore[nftID] does not exist:
-        raise Exception("NFT substore entry does not exist")
-
-    if isNFTEscrowed(nftID):
-        raise Exception("NFT is escrowed to another chain")
-
-    if getChainID(nftID) not in [OWN_CHAIN_ID, receivingChainID]:
-        raise Exception("NFT must be native to either the sending or the receiving chain")
-
     if messageFeeTokenID != Interoperability.getMessageFeeTokenID(receivingChainID):
```

Comment:
Maybe this is slightly out of the scope for this PR, but I don't think we need the parameter `messageFeeTokenID` really. It appears only in verification and not in the execution, so it does not contribute anything meaningful to the function. I would just remove it.

Reply:
Done in [this commit](https://github.com/LiskHQ/lips/pull/477/commits/d7c4ff4410cc66f66825d737b6d251feb870f37d).

Reply:
... and yes, nice catch, thanks!